Raven.Client.Lightweight/Document/BulkInsertOperation.cs

```diff
@@ -105,6 +107,8 @@ public void Store(object entity, string id)
 
         public void Store(RavenJObject document, RavenJObject metadata, string id, int? dataSize = null)
         {
+            id = string.IsNullOrWhiteSpace(id) ? Guid.NewGuid().ToString() : id.Trim();
```

Comment:
No I did not. I made this issue after a mail we receive 
https://groups.google.com/forum/#!topic/ravendb/gm-DFrPfu-s.
Didn't see where we add spaces to id in my checks